yesql.js

```diff
@@ -25,6 +25,11 @@ const readSqlFiles = (dir, options = {}) => {
 
 const pg = (query, options = {}) => {
   return (data = {}) => {
+
+    // sanitize comments
+    query = query.replace(/--[^\n]*$/gm, '')
+    query = query.replace(/\/\*[^\*\/]*\*\//gm, '')
```

Comment:
This failed cases where the comment contained a `*` e.g. this style would not work
```
/**
 * Some comment
 */
```

Found a solution in #27 that works with that too.